include/class.file.php

```diff
@@ -240,7 +240,9 @@ static function _genUrlSignature($id, $key, $signature, $expires) {
     }
 
     function download($disposition=false, $expires=false) {
-        $disposition = ($disposition && strcasecmp($disposition, 'inline') == 0
+        $thisstaff = StaffAuthenticationBackend::getUser();
+        $disposition = ((($disposition && strcasecmp($disposition, 'inline') == 0)
+              || $thisstaff->getImageAttachmentView() === 'New-Tab')
```

Comment:
Wouldn't this crash when you don't have an authenticated staff - e.g file view from Client Portal?

Reply:
lol it did, updated now